### ISSUE TITLE
Gts bugged with prices

### ISSUE THREAD

**Issue by jantjewieger** [SmashMC-Development/Bugs-and-Issues#980]
**Describe the Bug**
Whenever u pot a non HA and non-Shiny pokémon on the gts it cannot go for more than 20K. Bp's that normally have a price (unbreedable) for like 80K are now getting sold for 20K (breedable also getting sold for 20K to ruin eco).

**To Reproduce**
1. Get a BP
2. Sell it on the gts by doing /gts
3. try to set the price higher than 20K
4. Price gets put back to 20K automatically
5. U can thus not sell for more than 20K via /gts

**Servers with the Bug**
Groudon
Zekrom
Kyurem
Kyogre
Rayquaza

So basically everywhere but Reshiram

**Expected behavior**
I would expect the prices to be changed so that eco's dont get ruined by people underselling

**Screenshots**
NA

**Additional context**
NA

**Comment by Rasgnarok** [comment]
https://github.com/SmashMC-Development/Bugs-and-Issues/issues/976

**Comment by Rasgnarok** [comment]
https://github.com/SmashMC-Development/Bugs-and-Issues/issues/962

**Comment by Rasgnarok** [comment]
https://github.com/SmashMC-Development/Bugs-and-Issues/issues/836

**Comment by Superfire96** [comment]
Was fixed and minimum pricings were added to the gts
